Gems/AtomLyIntegration/EditorModeFeedback/Code/Source/EditorModeFeedbackFeatureProcessor.h

```diff
@@ -44,6 +44,9 @@ namespace AZ
             // AZ::TickBus overrides ...
             void OnTick(float deltaTime, AZ::ScriptTimePoint time) override;
 
+            //! Enable or disable the rendering editor mode feedback
```

Comment:
uber nit: Missing period at EOL

Reply:
I think we usually don't add period in the end if the comment is just one line. Although I will add it since it's not consistent in this file.